### PR TITLE
BD-1215: Revamp Content Card customization docs

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

> Adds information on subclassing vs complete view controller customization 

> Adds carousel implementation section to iOS and Android customization guides

Closes #**BD-1215**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/content_cards/customize.md [hunk 1]
```diff
@@ -1,29 +1,200 @@
...
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td><b>Effort</b></td>
+    <td>Low</td>
+  </tr>
+  <tr>
+    <td><b>Developer work</b></td>
+    <td>0-1 hours</td>
```
Comment (malandr2) [inline _docs/_user_guide/message_building_by_channel/content_cards/customize.md:57]: I would get rid of the `Developer work` section. There's a lot of variables to determine the work and would keep it general in the docs. Could refactor it from `0-1` to `Low`? But I wouldn't make any promises because we each customer is different. 

Same thought for the "Walk" and "Run" tab

File: _docs/_user_guide/message_building_by_channel/content_cards/customize.md [hunk 2]
```diff
@@ -1,29 +1,200 @@
...
+    word-break: break-word;
+}
+</style>
+
+{% tabs %}
+{% tab Crawl %}
+
+![Sample finance app showing Captioned Image and Banner Content Cards]({% image_buster/assets/img_archive/cc_pyrite_crawl.png %}){: style="max-width:35%;float:right;margin-left:15px;border:none;"}
+
+Best suited for teams with limited developer resources, the Crawl approach relies solely on out-of-the-box [Content Card templates]({{site.baseurl}}/user_guide/message_building_by_channel/content_cards/creative_details/) to get you up and running with Content Cards in as little as one hour.
```
Comment (malandr2) [inline _docs/_user_guide/message_building_by_channel/content_cards/customize.md:39]: I would reword "little as as one hour" to "less than 5 lines of code"

File: _docs/_user_guide/message_building_by_channel/content_cards/customize.md [hunk 3]
```diff
@@ -1,29 +1,200 @@
...
+{% endalert %}
+
+{% endtab %}
+{% tab Walk %}
+
+![Alt text]({% image_buster/assets/img_archive/cc_pyrite_walk.png %}){: style="max-width:35%;float:right;margin-left:15px;border:none;"}
+
+A hybrid approach to implementation, the Walk approach involves both marketing and developer teams pitching in to match your app or site's branding. 
+
+During the implementation process, developers write custom code to match the look and feel of Content Cards to that of your brand. This includes font type, font size, rounded corners, and colors. This approach still uses the out-of-the-box Content Cards, however, template styling is handled programmatically by your developers with subclassing.
```
Comment (malandr2) [inline _docs/_user_guide/message_building_by_channel/content_cards/customize.md:93]: I would get rid of "with subclassing" because this isn't a dev guide doc the concept of subclassing won't necessarily help the audience here

File: _docs/_user_guide/message_building_by_channel/content_cards/customize.md [hunk 4]
```diff
@@ -1,29 +1,200 @@
...
+</table>
+
+{% endtab %}
+{% tab Run %}
+
+![Alt text]({% image_buster/assets/img_archive/cc_pyrite_run.png %}){: style="max-width:35%;float:right;margin-left:15px;border:none;"}
+
+With the Run approach, developers take the lead with full control of the Content Card user experience. Custom code dictates what the cards will look like, how they behave, and how they interact with other messaging channels (e.g., triggering a Content Card based on a push notification). 
+
+The Braze SDK still handles the on-click behavior, order, and analytics.That means that marketers can still access valuable Content Card metrics in the Braze dashboard—like impressions, clicks, and dismissals.
```
Comment (malandr2) [inline _docs/_user_guide/message_building_by_channel/content_cards/customize.md:143]: In the "Run" tab, the Braze SDK does not handle the on-click behavior, order and analytics. This is reflected correctly in the table below. Could say something something those traits should be handled manually programmatically by the developer.

**Review by malandr2 — COMMENTED** [review]
Looking good! Repurposing the crawl-walk-run into docs is going to help tremendously

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md [hunk 5]
```diff
@@ -243,3 +247,55 @@ func contentCardTableViewController(_ viewController: ABKContentCardsTableViewCo
...
+
+For information on the analytics methods, refer to [Card methods]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/content_cards/data_model/#card-methods). 
+
+{% alert note %}
+The same page also details the different properties inherited from our generic Content Card model class, which you may find useful during your view implementation.
+{% endalert %}
+
+#### Step 3: Create a Content Card listener
+
+Create a [Content Card listener]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds/#step-2-set-up-a-content-card-listener) that is responsible for listening for the arrival of Content Cards, and implement conditional logic to display a specific number of cards in the carousel at any one time. By default, Content Cards are sorted by created date (newest first), and a user sees all cards they are eligible for.
```
Comment (malandr2) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md:283]: A "listener" is an Android concept I would change that word to "observer" for iOS

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md [hunk 6]
```diff
@@ -656,6 +658,53 @@ This section applies to integrations which use the Braze SDK's default Content C
 
 {% include archive/android/gifs.md channel="Content Cards" %}
 
+## Use case: Carousel view
+
+![Sample news app showing carousel of content cards in an article]({% image_buster/assets/img_archive/cc_politer_carousel.png %}){: style="max-width:35%;float:right;margin-left:15px;border:none;"}
```
Comment (malandr2) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md:663]: I would update the image to have an "Android" looking device be part of the screenshot, rather than an iOS device.

Comment (bre-fitzgerald) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md:663]: Ahhh, good point—will do!

File: _docs/_user_guide/message_building_by_channel/content_cards/customize.md (hunk 4 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/message_building_by_channel/content_cards/customize.md:143]: Thanks, @malandr2! Is it still correct to say that once the developers handle it programmatically, those metrics will be available in the dashboard for marketers?

Comment (bre-fitzgerald) [reply _docs/_user_guide/message_building_by_channel/content_cards/customize.md]: Resolved via slack

File: _docs/_user_guide/message_building_by_channel/content_cards/customize.md [hunk 7]
```diff
@@ -1,29 +1,188 @@
...
+### Use cases
+
+- Multiple Content Card feeds, such as adding a newsfeed, notification center, or promotions tab.
+- Display Content Cards in an existing feed.
+- Display Content Cards in a carousel view.
+- Use a Content Card to capture user information.
+- Trigger Content Cards based on other messaging channels.
+
+{% alert tip %}
+Check out the [sample use cases]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/content_cards/implementation_guide/#sample-use-cases) outlined in our Advanced Content Card implementation guide to get an idea of what you can do with this approach. Note that while the documentation is specific to iOS, the fundamentals are applicable to all operating systems.
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/content_cards/customize.md:181]: Hey Bre, I'd also link the Android version of the advanced implementation guide:
https://www.braze.com/docs/developer_guide/platform_integration_guides/android/content_cards/implementation_guide/

Comment (bre-fitzgerald) [reply _docs/_user_guide/message_building_by_channel/content_cards/customize.md:181]: Ah—thanks I didn't realize we had one! Will do 👍

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! So happy to get this content in here. Also, the formatting is really great, super clear, and easy to digest. Left one minor edit.

**Review by KellieHawks — APPROVED** [review]
Looks great! thanks for those edits!
